### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
#### What is the purpose of this change?

Spelling fixes

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [ ] I have added tests for all changes in this PR if appropriate. -- check-spelling could catch things like this...
- [ ] I have added documentation for the changes if appropriate.
- [ ] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages). -- I'll wait until the general changes are deemed acceptable before adjusting to match
- [ ] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

**Review by albertony — CHANGES_REQUESTED** [review]
Amazing! 😲 

Surprising how many typos, and especially double words like "the the", it is possible to write - and miss in reviews etc.. I'm sure there are a couple only in this short comment! 🙈 

Would be interesting to see if we could get the check-spelling GitHub Action up and running..?

I went through the current set of changes, found only a couple of changes that are in generated code, expected a lot more of that. Great work.

File: backend/dropbox/dropbox.go [hunk 1]
```diff
@@ -268,7 +268,7 @@ default based on the batch_mode in use.
 			Advanced: true,
 		}, {
 			Name:     "batch_commit_timeout",
-			Help:     `Max time to wait for a batch to finish comitting`,
+			Help:     `Max time to wait for a batch to finish committing`,
```
Comment (jsoref) [inline backend/dropbox/dropbox.go:271]: This is the change that is reflected in the auto-generated content...

File: docs/content/dropbox.md [hunk 2]
```diff
@@ -409,7 +409,7 @@ Properties:
 
 #### --dropbox-batch-commit-timeout
 
-Max time to wait for a batch to finish comitting
+Max time to wait for a batch to finish committing
```
[the remark this answers is not in the document]
Comment (jsoref) [reply docs/content/dropbox.md:412]: This corresponds to: https://github.com/rclone/rclone/pull/6380/commits/df1cfe90fd3c19c080eee7053699719d9224a51a#r949548776

[the remark this answers is not in the document]
Comment (albertony) [reply docs/content/dropbox.md:412]: Right.. So you have fixed the sources of the two autogenerated changes. Good.
Usually we do not commit changes in the .md files then, though I'm not sure how much it matters.

**Comment by jsoref** [comment]
@albertony: are you ok w/ the changes as is? If so, I can squash and write a proper commit message.

And yes, I'm more than happy to make a PR for the workflow. (Also happy to chat/demo it if that's helpful.)

**Comment by albertony** [comment]
👍 But lets wait to see what @ncw has to say.

**Comment by albertony** [comment]
Is this PR result of running your tool only, or did you do some manual corrections as well?

Integration with golangci-lint would be nice, since we already use this to run a suite of "code quality" checks. Have you considered that?

I see there is an alternative called "misspell" which is in fact supported with golangci-lint. I just tried it real quick: Enabled it and ran it without any additional config. It found 34 errors, all in .go files. According to docs, by default it only checks comments in .go files, and must be configured to check entire source files for it to detect errors in strings. But the golangci-lint integration is always checking entire source ("Treat Go source files as a plain text by misspell: it allows detecting issues in strings, variable names, etc.", [ref](https://github.com/golangci/golangci-lint/blob/master/CHANGELOG.md#june-2019)). Strings are way more important than comments, at least in rclone, to get all the help texts correct, which are also used for generated documentation. And also the user documentation .md files are important. It would be easy for us to enable misspell, and it would be helpful enough, although quite limited results. Also; latest commit in the github repo is from 2018 so probably not much hope for improvements..

**Comment by jsoref** [comment]
The tool more or less only flags, it doesn't correct. But I believe that everything here was based on its output.

If you want the tool to only look at certain files, you can use `excludes.txt` and `only.txt` to tailor which files it looks at. (Note that this PR doesn't include the tool at all, it's purely)

I'm looking into generating SARIF output.

**Comment by albertony** [comment]
I think this is good.

> If so, I can squash and write a proper commit message.

If you do that, I will merge it. :)

**Comment by albertony** [comment]
I will merge this now (I didn't notice you had squashed it until now). Thank you!

**Pull request merged**